### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NullableInt**
Found a small error in README.md

## Description
Found a small error in README.md

## Motivation and Context
The motivation is that I found it, and thought it could be fixed.

## How Has This Been Tested?
Didn't test at all!

## Screenshots (if appropriate):
Not this time

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by NullableInt** [comment]
> 
> 
> Shouldn't it be | `@` | `~/src/App/Documentation/utils`-folder (example `import { DocToc } from "@docutils";`)? Nice find btw!

Maybe! The line above shows us an example of the usage of `@` for importing.

**Comment by VikingTristan** [comment]
My bad, I forgot these were listed in the readme when I updated imports a while ago.
Current imports work like this (from .babelrc)
image

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=5387638&bid=17664316)

File: README.md [hunk 1]
```diff
@@ -62,12 +62,13 @@ it, **PayEx**. Check out the `package.json` file for additional scripts.
...
-| Shortcut | Description |
-|:--------:|:------------|
-|   `~`    | root of the project (example `import package from "~/package";`)
-|   `$`    | `~/src`-folder (example `import dg from "$/scripts";`)
-|   `@`    |  `~/src/App/components`-folder (example `import Alert from "@components/Alert";`)
-|   `#`    | `~/src/App/Documentation/utils`-folder (example `import { DocToc } from "@docutils";`)
+|   Shortcut    | Description                                                                            |
+| :-----------: | :------------------------------------------------------------------------------------- |
+|      `~`      | root of the project (example `import package from "~/package";`)                       |
+|      `$`      | `~/src`-folder (example `import dg from "$/scripts";`)                                 |
```
Comment (VikingTristan) [inline README.md:68]:
```suggestion
|      `@src`      | `~/src`-folder (example `import dg from "@src/scripts";`)                                 |
```

File: README.md [hunk 2]
```diff
@@ -62,12 +62,13 @@ it, **PayEx**. Check out the `package.json` file for additional scripts.
...
+| `@components` | `~/src/App/components`-folder (example `import Alert from "@components/Alert";`)       |
+|  `@docutils`  | `~/src/App/Documentation/utils`-folder (example `import { DocToc } from "#docutils";`) |
```
Comment (VikingTristan) [inline README.md:70]:
```suggestion
|  `@docutils`  | `~/src/App/Documentation/utils`-folder (example `import { DocToc } from "@docutils";`) |
```

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=5387638&bid=17664316)

File: README.md [hunk 3]
```diff
@@ -62,12 +62,13 @@ it, **PayEx**. Check out the `package.json` file for additional scripts.
...
-| Shortcut | Description |
-|:--------:|:------------|
-|   `~`    | root of the project (example `import package from "~/package";`)
-|   `$`    | `~/src`-folder (example `import dg from "$/scripts";`)
-|   `@`    |  `~/src/App/components`-folder (example `import Alert from "@components/Alert";`)
-|   `#`    | `~/src/App/Documentation/utils`-folder (example `import { DocToc } from "@docutils";`)
+|   Shortcut    | Description                                                                            |
+| :-----------: | :------------------------------------------------------------------------------------- |
+|      `~`      | root of the project (example `import package from "~/package";`)                       |
+|    `$src`     | `~/src`-folder (example `import dg from "@src/scripts";`)                              |
```
Comment (VikingTristan) [inline README.md:68]:
```suggestion
|    `@src`     | `~/src`-folder (example `import dg from "@src/scripts";`)                              |
```

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=5387638&bid=17664316)

**Comment by NullableInt** [comment]
I can not press the merge button, so someone with `write access` has to.

**Pull request merged**